description.md

```diff
@@ -19,6 +19,35 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Drills
 
+Drills are educational activities designed to reinforce learning through repetitive practice.
+While quizzes are assessments aimed at testing understanding, drills involve repeated practice of a specific skill or concept.
+The primary goal of drills is to promote mastery and internalisation of skills or knowledge.
+By repeating tasks or exercises, students can reinforce their understanding and enhance their ability to apply learned concepts effectively.
+However, it's important to balance drills with other types of learning activities to maintain student engagement and prevent monotony.
+For example, drills can be combined with homework assignments, group discussions, and other activities to provide a more varied learning experience.
+The difference is that while drills are repetitive, simple, and focused on a single skill or concept, homework activities can be more complex and open-ended.
+
+Some key considerations when designing drills include:
+
+- **Skill acquisition**: By providing repetitive practice, drills help students develop accuracy and speed.
+- **Reinforcement**: Drills should be used to reinforce skills and knowledge that have already been taught.
+- **Building foundations**: By practicing fundamental skills and concepts repeatedly, students can develop a solid base of knowledge, which can support their learning in more complex areas.
+- **Confidence**: Drills can boost student confidence as they see their progress through repeated practice.
+- **Engagement**: Start with simple drills and gradually increase the difficulty level to prevent students from becoming overwhelmed.
+
+To overcome this limitation, some key considerations include:
```

Comment:
"key consideration" is repeated. Maybe also clarify which limitation are you talking about.